payments/schema/mutations.py

```diff
@@ -640,6 +640,19 @@ class Input:
     sent_orders = graphene.List(graphene.NonNull(graphene.ID), required=True)
     failed_orders = graphene.List(graphene.NonNull(FailedOrderType), required=True)
 
+    @classmethod
+    def prepare_for_resending(cls, order):
+        if order.status == OrderStatus.ERROR or order.lease.status == LeaseStatus.ERROR:
+            order.set_status(
+                OrderStatus.WAITING,
+                comment=f"{today()}: {_('Cleanup the invoice to attempt resending')}\n",
+            )
+        elif order.status == OrderStatus.CANCELLED:
+            order.set_status(
+                OrderStatus.WAITING,
+                comment=f"{today()}: {_('Resend cancelled invoice')}\n",
+            )
+
```

Comment:
This should go to `payments/utils.py` to make it a reusable util :)

Reply:
fixed